src/uu/cp/src/cp.rs

```diff
@@ -866,6 +869,44 @@ fn copy_source(
     }
 }
 
+pub fn normalize_path<P: AsRef<Path>>(path: P) -> PathBuf {
+    let mut components = path.as_ref().components().peekable();
+    let mut ret = if let Some(c @ Component::Prefix(..)) = components.peek().cloned() {
+        components.next();
+        PathBuf::from(c.as_os_str())
```

Comment:
You might be able to avoid `cloned()` by reordering this branch to something like:
```rust
let buf = PathBuf::from(c.as_os_str());
components.next();
buf
```